components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/nashorn/JsNashornClaims.java

```diff
@@ -245,6 +246,11 @@ private void setLocalUserClaim(String claimUri, Object claimValue) {
             claimUriMap.put(claimUri, String.valueOf(claimValue));
             ((AbstractUserStoreManager) userRealm.getUserStoreManager())
                     .setUserClaimValuesWithID(authenticatedUser.getUserId(), claimUriMap, null);
+        } catch (UserStoreClientException e) {
```

Comment:
Was the client exception has the same cause as in parent userStoreException

Reply:
Both UserStoreException and UserStoreClientException are thrown for different causes from the methods within the try block.

Only the client exceptions will not have error logging, for UserStoreExceptions error logging is needed.